NEWS

```diff
@@ -16,6 +16,7 @@ What's New in libchewing (unreleased)
   - New boolean config option `chewing.sort_candidates_by_frequency` can be
     set to enable sorting candidates by frequency.
   - Rust type SyllableSlice is removed from public interfaces.
+  - Provide libchewing as a Swift package.
```

Comment:
- Provide libchewing as a Swift package with the exposures of C_based APIs only.

Reply:
Make sense. For me `libchewing` refers only to the C API but your edit makes it more clear for people unfamiliar with the history of libchewing.